packages/swap/prisma/schema.prisma

```diff
@@ -417,23 +418,19 @@ model StateChainError {
   @@schema("public")
 }
 
-model QuoteResult {
-  id                 Int           @id @default(autoincrement())
-  srcAsset           InternalAsset
-  destAsset          InternalAsset
-  depositAmount      Decimal       @db.Decimal(30, 0)
-  quoterIntermediate Decimal?      @db.Decimal(30, 0)
-  quoterOutput       Decimal       @db.Decimal(30, 0)
-  quoterDuration     Int
-  quoterFees         Json?
-  poolIntermediate   Decimal?      @db.Decimal(30, 0)
-  poolOutput         Decimal       @db.Decimal(30, 0)
-  poolDuration       Int?
-  poolFees           Json?
-  createdAt          DateTime      @default(now())
-  version            Int
-
-  @@index([srcAsset, destAsset])
+model Quote {
```

Comment:
wondering if it would be useful to know if it was a boost/dca quote.

Reply:
without checking the channel

Reply:
yeah true, might be convenient - will add two booleans

Reply:
will do this in a separate pr because i already built on this